### DISCUSSION TITLE
Users overwrite each other through websocket when connection is reestablished

### DISCUSSION THREAD

**Discussion by JustDoItSascha** [outline/outline#5791] [Self Hosting]
**To Reproduce**
My colleagues and I discovered a bug. I open Outline and a document. Then I close my macbook. My colleague changes something in the same document. Then I open my macbook again and it seems that my outline connects via websocket to the server and overwrites the changes my colleague makes. His changes are deleted.

**Expected behavior**
I would assume that outline compares the versions and changes and does not overwrite things others wrote while my connection to outline was not online.

**Screenshots**
If applicable, add screenshots or videos to help explain your problem.

**Outline (please complete the following information):**
- Install: self hosted
- Version: 0.71.0

**Desktop (please complete the following information):**
 - Browser chrome

**Comment by sncat** [comment]
As mentioned in this issue, the same overwrite seems to be happening.
When it happened in my environment, even the edit history was overwritten and no rollback was possible...

https://github.com/outline/outline/issues/5830
